contracts/interfaces/IPaymaster.sol

```diff
@@ -30,7 +30,7 @@ interface IPaymaster {
      * @return validationData - Signature and time-range of this operation, encoded the same as the return
      *                          value of validateUserOperation.
      *                          <20-byte> sigAuthorizer - 0 for valid signature, 1 to mark signature failure,
-     *                                                    otherwise, an address of an "authorizer" contract.
+     *                                                    other values are invalid for paymaster.
```

Comment:
It is probably better to remove any mention of the `<20-byte> sigAuthorizer` as well, as it is not relevant for the Paymaster.